packages/ignite/src/plugins/default/commands/add.js

```diff
@@ -112,31 +112,37 @@ Examples:
 
   // ok, are we ready?
   try {
-    // bring the ignite plugin to life
-    const pluginModule = require(modulePath)
-
-    // set the path to the current running ignite plugin
-    ignite.setIgnitePluginPath(modulePath)
-
-    // now let's try to run it
-    try {
-      await pluginModule.add(context)
-
-      // We write the toml changes
-      const combinedGenerators = Object.assign({}, currentGenerators, proposedGenerators)
-      const updatedConfig = R.assocPath(['ignite', 'generators'], combinedGenerators, context.config)
-      const localToml = `${process.cwd()}/ignite/ignite.toml`
-      filesystem.write(localToml, json2toml(updatedConfig))
-
-      // Sweet! We did it!
-      success('🍽  time to get cooking!')
-      process.exit(exitCodes.OK)
-    } catch (err) {
-      // it's up to the throwers of this error to ensure the error message is human friendly.
-      // to do this, we need to ensure all our core features like `addModule`, `addComponentExample`, etc
-      // all play along nicely.
-      error(err.message)
-      process.exit(exitCodes.PLUGIN_INSTALL)
+    if (filesystem.exists(modulePath) === 'file') {
+      // bring the ignite plugin to life
+      const pluginModule = require(modulePath)
+
+      // set the path to the current running ignite plugin
+      ignite.setIgnitePluginPath(modulePath)
```

Comment:
let's do a validation step of making sure they have add AND remove.  Otherwise exit, bc shit fucked up.